src/main/java/com/google/api/codegen/config/PackageMetadataConfig.java

```diff
@@ -381,6 +396,10 @@ public ReleaseLevel apply(@Nullable String releaseLevelName) {
    */
   private static <V> Map<TargetLanguage, V> buildMapWithDefault(Map<String, V> inputMap) {
     Map<TargetLanguage, V> outputMap = new HashMap<>();
+    if (inputMap == null) {
```

Comment:
Please see the previous comment, same applies here.

Reply:
ditto :)